### ISSUE TITLE
Crashes While Generating Covers

### ISSUE THREAD

**Issue by LillyenRose** [weto91/kindle-cover-fixer#15]
**Describe the bug**
When generating the covers for the list of books, the program will pause for a second and then crash and completely close the application. Each time it will reach a certain point and then crash. Have tried to generate covers on two different computers and it crashed (albeit at different points) on both computers.

**To Reproduce**
1. Click on "Find Books" 
2. Click on "Generate Covers"
3. Covers start generating
4. Application pauses and then crashes (closes)

**Expected behavior**
Figured it would generate the covers for the list of books and allow me to send to my device (Kindle Scribe)

**Desktop (please complete the following information):**
 - OS: Windows 10 and Windows 11
 - Browser Google Chrome
 - Version 2.5

**Additional context**
 Library is 419 books. On one computer it got to 217 and then crashed. On the other it got to 73 then crashed. I tried to delete the book it crashed on (for the second computer) and that did not help, instead making it crash on book 72. When the application pauses the mouse icon on the screen gets a wheel icon (like its loading something) and then the application will close suddenly (or crashes).

**Comment by weto91** [comment]
Also check if all the books in the library have a cover. Currently there is a bug in the application in which if there is no cover, it closes (We are fixing it right now)

**Comment by weto91** [comment]
> > Also check if all the books in the library have a cover. Currently there is a bug in the application in which if there is no cover, it closes (We are fixing it right now)
> 
> Thank you, I have the same error with some books in Calibre are no covers.

Yes, currently the application crashes when there are books without covers (We are developing the fix). However, those books that do not have covers cannot be processed. I recommend you add cover from caliber to all of them, transfer them again and try the application again.

All the best.

**Comment by LillyenRose** [comment]
Here is the Log...
[Kindle_Cover_Fixer.log](https://github.com/weto91/kindle-cover-fixer/files/13482668/Kindle_Cover_Fixer.log)
I hope this is the correct file. 
I did check my covers and everything has a cover. Thanks for the help!!! I will go ahead and check covers again since i have so many books in the library

**Comment by weto91** [comment]
> Here is the Log... [Kindle_Cover_Fixer.log](https://github.com/weto91/kindle-cover-fixer/files/13482668/Kindle_Cover_Fixer.log) I hope this is the correct file. I did check my covers and everything has a cover. Thanks for the help!!! I will go ahead and check covers again since i have so many books in the library

Hello!,
Please download version 2.6, I think your problem is solved in this version.

Try it and tell me!

Regards.

**Comment by LillyenRose** [comment]
> > Here is the Log... [Kindle_Cover_Fixer.log](https://github.com/weto91/kindle-cover-fixer/files/13482668/Kindle_Cover_Fixer.log) I hope this is the correct file. I did check my covers and everything has a cover. Thanks for the help!!! I will go ahead and check covers again since i have so many books in the library
> 
> Hello!, Please download version 2.6, I think your problem is solved in this version.
> 
> Try it and tell me!
> 
> Regards.

Hey! This worked perfectly!! Thank you so much!! The program crashed a few times upon opening, but i found out that was because of my Calibre needing to be opened for some reason. Once i opened it, and then restarted yours it worked without a problem!! Thank you so much 1) for the quick replies and help and 2) for the amazing program that fixed my covers (which i have been fighting with for a LONG time)!!!
